### PR TITLE
Apply changes from original cefglue repo

### PR DESCRIPTION
**Pull request by cangosta**

### CODE REVIEW

File: CefGlue.Common/CommonCefClient.cs [hunk 1]
```diff
@@ -117,6 +117,10 @@ protected override bool OnProcessMessageReceived(CefBrowser browser, CefFrame fr
                 _owner.HandleException(e);
                 return false;
             }
+            finally
```
Comment (joaompneves) [inline CefGlue.Common/CommonCefClient.cs:120]: using (message)

File: CefGlue.BrowserProcess/Handlers/RenderProcessHandler.cs [hunk 2]
```diff
@@ -29,14 +29,22 @@ protected override void OnWebKitInitialized()
 
         protected override bool OnProcessMessageReceived(CefBrowser browser, CefFrame frame, CefProcessId sourceProcess, CefProcessMessage message)
         {
-            WithErrorHandling(() =>
+            try
```
Comment (joaompneves) [inline CefGlue.BrowserProcess/Handlers/RenderProcessHandler.cs:32]:
```suggestion
            using (message)
```

Comment (cangosta) [reply CefGlue.BrowserProcess/Handlers/RenderProcessHandler.cs]: done

File: CefGlue.Common/CommonCefClient.cs (hunk 1 above)
Comment (cangosta) [reply CefGlue.Common/CommonCefClient.cs]: done

File: CefGlue.Common/CommonCefClient.cs [hunk 3]
```diff
@@ -109,8 +109,11 @@ protected override bool OnProcessMessageReceived(CefBrowser browser, CefFrame fr
         {
             try
             {
-                _messageDispatcher.DispatchMessage(browser, frame, sourceProcess, message);
-                return base.OnProcessMessageReceived(browser, frame, sourceProcess, message);
+                using (message)
+                {
```
Comment (joaompneves) [inline CefGlue.Common/CommonCefClient.cs:113]: dont need the { }
```suggestion
                using (message)
```

**Review by joaompneves — APPROVED** [review]

**Pull request merged**